sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/cryptography/cryptography_client.hpp

```diff
@@ -0,0 +1,288 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief A client used to perform cryptographic operations with Azure Key Vault keys.
+ *
+ */
+
+#pragma once
+
+#include <azure/keyvault/common/internal/keyvault_pipeline.hpp>
+
+#include <azure/core/io/body_stream.hpp>
+
+#include "azure/keyvault/keys/cryptography/cryptography_client_options.hpp"
+#include "azure/keyvault/keys/cryptography/cryptography_provider.hpp"
+#include "azure/keyvault/keys/cryptography/encrypt_parameters.hpp"
+#include "azure/keyvault/keys/cryptography/encrypt_result.hpp"
+#include "azure/keyvault/keys/cryptography/remote_cryptography_client.hpp"
+#include "azure/keyvault/keys/cryptography/sign_result.hpp"
+#include "azure/keyvault/keys/cryptography/wrap_result.hpp"
+
+#include <memory>
+#include <string>
+
+namespace Azure {
```

Comment:
Why is Cryptography under Keys.  I would expect this to be in Azure::Core::Crptography or Azure::Security::Cryptography

Reply:
It contains the Cryptography client for Keys operations which are performed on Key Vault Keys service.
That´s the path on other languages as well. :)

There are some stuff like the Hashing support that we can move to Core but for now I left it on Key Vault common due to Core going GA on May.  The implementation can be moved to Core someday in the future

Reply:
@RickWinter that's where it is in all the other languages. It's specific to keys, and - while it predates my time on this team - nesting it under another namespace was to reduce the "clutter" of too many types in a namespace.